helm-deploy-test/tasks/cf-teardown.sh

```diff
@@ -11,9 +11,9 @@ UAA_NAMESPACE=uaa
 set +o allexport
 
 for namespace in "$CF_NAMESPACE" "$UAA_NAMESPACE" ; do
-    kubectl delete namespace "${namespace}" || true
+    kubectl delete namespace "${namespace}"
```

Comment:
I removed the `|| true` here.. since kube deletions are async, if this fails, something may actually be wrong (api server inaccessible, corrupt kube config). This isn't retried, so we shouldn't `|| true` it.

Reply:
The question then is, _why_ isn't this retried? Sometimes kube API access falls over temporarily (because the machine is being bogged down by the SCF trying to start).

Reply:
if we don't use `|| true` then the scrip just exit and pipeline takes it as failure. Which is not true.

Reply:
@mook-as this script only runs after passing acceptance tests, so SCF shouldn't be trying to start at that point. I don't see why we'd specifically want a retry here over all the other `kubectl` invocations in [cf-deploy.sh](https://github.com/SUSE/cf-ci/blob/53b8ea561d5f4b1ba492cef10c761ef2df7ca055/helm-deploy-test/tasks/cf-deploy.sh)

Reply:
SCF running could cause things to be slow too, if we had a bug there (but then it'd probably fail CATS).

I _guess_ that's okay…

Reply:
I don't think the ability of apiserver to respond should be affected by existing workloads (scf in this case), since the kubelet service isn't running on the master.